en/config/compass.md

```diff
@@ -3,12 +3,21 @@
 The compass calibration process configures all connected internal and external [magnetometers](../gps_compass/README.md).
 *QGroundControl* will guide you to position the vehicle in a number of set orientations and rotate the vehicle about the specified axis.
 
-> **Note** If you are using an external magnetometer/compass (e.g. a compass integrated into a GPS module) make sure you mount the external compass on your vehicle properly and connect it to the autopilot hardware. Instructions for connecting your GPS+compass can be found in [Basic Assembly](../assembly/README.md) for your specific autopilot hardware. Once connected, QGroundControl will automatically detect the external magnetometer. 
+> **Note** If you are using an external magnetometer/compass (e.g. a compass integrated into a GPS module) make sure you mount the external compass on your vehicle properly and connect it to the autopilot hardware.
+  Instructions for connecting your GPS+compass can be found in [Basic Assembly](../assembly/README.md) for your specific autopilot hardware.
+  Once connected, *QGroundControl* will automatically detect the external magnetometer.
+
+<span></span>
+> **Tip** You will need to calibrate your compass on first use, and you may need to recalibrate it if the vehicles is ever exposed to a very strong magnetic field, or if it is used in an area with abnormal magnetic characteristics.
+  Indications of a poor compass calibration include multicopter circling during hover, or veering off-path when attempting to fly straight.
```

Comment:
Maybe you want to mention 'toilet bowl'?

Reply:
Isn't toilet bowl circling during hover? "Definitions" on the Internet range from "rotate in place during hover" to "circling during hover". I would have thought a toilet was an inward spiral leading to being sucked to ground :-)

Reply:
@bkueng ^^^ Can you please confirm the correct definition of toilet bowling?

Reply:
It is starting to circling, starting off small usually, and getting larger until you have a fly-away. So you can say it's a spiral, but the altitude is usually constant.

Reply:
Thanks very much. Integrated.